integrationtest/wal_rollback_test.go

```diff
@@ -208,38 +212,13 @@ func checkObjects(t *testing.T, roleConfig map[string]interface{}, isClusterBind
 	}
 	bo := backoff.NewExponentialBackOff()
 	bo.MaxElapsedTime = maxWaitTime
+	// Don't actually back off, just keep retrying quickly to speed up the test.
+	bo.Multiplier = 1
```

Comment:
Should this just be a ConstantBackOff?

Reply:
Oh I didn't know that was there. Unfortunately it doesn't seem to support a maximum time, so it's quite convenient to continue using exponential backoff, despite the slight abuse of the intent.